package.json

```diff
@@ -36,7 +36,7 @@
     "express": "^4.17.1",
     "fast-deep-equal": "^2.0.1",
     "frappe-charts": "^2.0.0-rc13",
-    "frappe-datatable": "^1.15.4",
+    "frappe-datatable": "^1.16.0",
```

Comment:
`package-lock.json` too needs to be updated 😄

Reply:
(this reminds me, we need to use `npm ci` or whatever is yarn equivalent of that.

Reply:
@hrwX Please run `yarn add frappe-datatable` and it will update both the files.